pyiron/base/job/generic.py

```diff
@@ -1481,7 +1486,7 @@ def _run_if_finished(self, run_again=False):
             self.run()
```

Comment:
Can you implement `delete_existing_job here`?

Reply:
I'm not really sure if such an option should be available in `run`, because it doesn't work in the interactive mode, and we know that the changes in input won't be reflected in this case. Isn't it better to drop it altogether?

Reply:
But we use it, for example here https://github.com/pyiron/pyiron/blob/master/pyiron/interactive/sxextoptint.py#L359

Reply:
But isn't it in conflict with `job.set_input_to_read_only()`? I don't understand when exactly the input can be changed and when it cannot.

Reply:
They basically only change the structure.